AutoMad/autoMad.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 22 13:14:59 2024
+
+@author: dragon
+"""
+
+import random
+import re
+import wordGrabber
+import madLibFeeder
+import madDictionary
+import nltk
+import pandas as pd
```

Comment:
It is good practice to create a [requirements.txt](https://pip.pypa.io/en/stable/reference/requirements-file-format/) file listing all the libraries you use that are not part of the standard library. This makes it much easier for someone else (or you on a different machine) to get started using the code.